src/Config.ts

```diff
@@ -73,13 +73,17 @@ export class Config {
 
     public readonly access: IConfigAccessControl = { };
 
-    public getRoomRule(roomIdOrAlias?: string) {
-        const aliasRule = this.roomRules.find((r) => r.room === roomIdOrAlias);
-        if (aliasRule && aliasRule.action === "deny") {
+    public getRoomRule(roomIdOrAlias: string) {
+        const rule = this.roomRules.find((r) => {
+            if (r.room === roomIdOrAlias) {
+                return true;
+            }
+            return (r.roomRegex && roomIdOrAlias.match(r.roomRegex));
```

Comment:
We have no interest in the return value. `.test()` may be faster and is more intentional.
```suggestion
            return (r.roomRegex && r.roomRegex.test(new RegExp(roomIdOrAlias)));
```